scalableminds/webknossos
# Systemd configuration

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Follow the steps described in the README
- => should start webknossos datastore module
- This uses the `run.sh` script from #5579, but you can easily swap executed command

### Issues:
- fixes #5547 

------
